hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_order_data_manager.py

```diff
@@ -0,0 +1,213 @@
+            quantity_remaining=os_update.quantity_remaining,
+            side=os_update.side,
+            fill_price=os_update.fill_price,
+            order_price=os_update.order_price,
+            client_order_i_d=client_order_id,
+            timestamp=time.time(),
+        )
+
+        updated = False
+        if client_order_id in order_statuses:
+            if order_statuses[client_order_id][-1].quantity_remaining != os_update.quantity_remaining:
+                order_statuses[client_order_id].append(order_status_info)
+                updated = True
+        else:
+            order_statuses[client_order_id] = [order_status_info]
+            updated = True
+
+        if updated:
+            remaining = order_status_info.quantity_remaining
+            released = order_status_info.quantity_released
+            if order_status_info.side == api.Side.S_BID:
+                remaining = round(remaining / order_status_info.order_price, 2)
+                released = round(released / order_status_info.order_price, 2)
+            HummingbotApplication.main_application().notify(
+                f"order type {order_status_info.order_status.name} | quantity released: {released} | "
+                f"quantity remaining: {remaining} | price:  {order_status_info.order_price} | "
+                f"side: {order_status_info.side.name} | id {client_order_id}")
+
+    def get_order_book(self, trading_pair: str) -> (Orderbook, float):
+        normalized_trading_pair = normalize_trading_pair(trading_pair)
+        if normalized_trading_pair not in self._order_books:
+            raise Exception(f"order book manager does not support ${trading_pair}")
+
+        ob_info = self._order_books[normalized_trading_pair]
+        return ob_info.latest_order_book, ob_info.timestamp
+
+    def get_price_with_opportunity_size(self, trading_pair: str, is_buy: bool) -> (float, float):
+        normalized_trading_pair = normalize_trading_pair(trading_pair)
+        if normalized_trading_pair not in self._order_books:
+            raise Exception(f"order book manager does not support ${trading_pair}")
+
+        ob_info = self._order_books[normalized_trading_pair]
+        return (
+            (ob_info.best_bid_price, ob_info.best_bid_size)
+            if is_buy
+            else (ob_info.best_ask_price, ob_info.best_ask_size)
+        )
+
+    def get_order_status(self, trading_pair: str, client_order_id: int) -> List[OrderStatusInfo]:
+        normalized_trading_pair = normalize_trading_pair(trading_pair)
+        if normalized_trading_pair not in self._markets_to_order_statuses:
+            raise Exception(f"order book manager does not support ${trading_pair}")
+
+        order_statuses = self._markets_to_order_statuses[normalized_trading_pair]
+        if client_order_id in order_statuses:
+            return order_statuses[client_order_id]
+
+        return []
+
+
+def normalize_trading_pair(trading_pair: str):
```

Comment:
is this just because of the hummingbot order type? (iirc it is). If so, just write a comment on why we are using it